packages/client/src/__tests__/tlog/verify/set.test.ts

```diff
@@ -101,6 +101,23 @@ describe('verifyTLogSET', () => {
     });
 
     describe('when the public key for the matching TLogInstance is not valid', () => {
+      describe('when public key has has no key', () => {
+        const tlogs: sigstore.TransparencyLogInstance[] = [
+          {
+            ...validTLog,
+            publicKey: {
+              rawBytes: undefined,
+              keyDetails: sigstore.PublicKeyDetails.PKIX_ECDSA_P256_SHA_256,
+              validFor: { start: undefined },
+            },
+          },
+        ];
+
+        it('returns false', () => {
+          expect(verifyTLogSET(entry, tlogs)).toBe(false);
+        });
+      });
+
```

Comment:
Covers a scenario (a key in the `trusted_root.json` which was missing the actual bytes of the key) which was not previously tested.